docs/model_config.md

```diff
@@ -0,0 +1,364 @@
+`threshold` parameter. If unset, an implicit threshold of 0.5 is used for
+binary classifiers.
+```
+parameters [
+  {
+    key: "threshold"
+    value: { string_value: "0.3" }
+  }
+]
+```
+
+### Performance parameters
+The FIL backend includes several parameters that can be tuned to optimize
+latency and throughput for your model. These parameters will not affect model
+output, but experimenting with them can significantly improve model
+performance for your specific use case.
+
+#### `use_experimental_optimizations`
+As of release 22.11, this parameter only affects CPU deployments. Setting it to
+true can significantly improve both latency and throughput. In the future,
+the current experimental optimizations will become default, and new
+performance optimizations will be trialed using this flag. Even these
+experimental optimizations are thoroughly tested before release, but they
+offer less of a stability guarantee than the default execution mode.
+```
+parameters [
+  {
+    key: "use_experimental_optimizations"
+    value: { string_value: "true" }
+  }
+]
+```
+
+#### `threads_per_tree` (GPU only)
+This parameter applies only to GPU deployments and determines the number
+of consecutive CUDA threads used to evaluate a single tree. While
+correctly tuning this value offers significant performance improvements,
+it is very difficult to determine the optimal value *a priori*.
+
+In general, servers under higher load or those receiving larger batches from
+clients will benefit from a higher value. On the other hand, more
+powerful GPUs typically see optimal performance with a somewhat lower value.
+
+To find the optimal value for your deployment, test under realistic traffic
+and experiment with powers of 2 from 1 to 32.
+```
+parameters [
+  {
+    key: "threads_per_tree"
+    value: { string_value: "4" }
+  }
+]
+```
+
+#### `storage_type` (GPU only)
+This parameter determines how trees are represented in device memory.
+Choosing `DENSE` will consume more memory but may sometimes offer
+performance benefits. Choosing `SPARSE` will consume less memory and may
+perform as well as or better than `DENSE` for some models. Choosing `AUTO`
+will apply a heuristic that defaults to `DENSE` unless it is obvious that
+doing so will consume significantly more memory.
+
+`SPARSE8` is an
+experimental format which offers an even smaller memory footprint than
+`SPARSE` and may offer better throughput/latency in some cases. You should
+thoroughly test your model's output with `SPARSE8` if you choose to use it.
+
+```
+parameters [
+  {
+    key: "storage_type"
+    value: { string_value: "SPARSE" }
+  }
+]
+```
+
+#### `algo` (GPU only)
+This parameter determines how nodes within a tree are organized in memory
+as well as how they are accessed during inference. `NAIVE` uses a
+breadth-first layout and is the only value which should be used with the
+`SPARSE` or `SPARSE8` storage types. `TREE_REORG` rearranges trees to improve
```

Comment:
It can, but it's about to be rendered unnecessary anyway by the new FIL implementation, so it's probably not worth it to add that logic to the FIL backend at this point.